backend/app/users/models.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from sqlalchemy import TIMESTAMP, Integer, String, text
+from sqlalchemy.orm import Mapped, mapped_column
+
+from ..models import Base
+
+
+class Users(Base):
```

Comment:
The convention we have been following is to add `DB` at the end so its not confused with the schema. So this should be `UsersDB`